packages/replay/src/bin.ts

```diff
@@ -39,6 +42,11 @@ program
   .option("--api-key <key>", "Authentication API Key")
   .action(commandUploadRecording);
 
+program
+  .command("launch <browser>")
```

Comment:
I'd like `launch <browser> <url>` but that's just me.

Reply:
we can make that happen